### ISSUE TITLE
Navigate To window automatically closing when VsVim is enabled

### ISSUE THREAD

**Issue by dnperfors** [jaredpar/VsVim#1815]
When I have VsVim enabled, the Navigate To functionality (Ctrl+,) of Visual Studio isn't really working. The 'search window' is opened, but while typing, it is closed automatically without waiting for Enter.

![navigateto](https://cloud.githubusercontent.com/assets/170992/17358397/1c3c0950-5962-11e6-8c89-19024397f235.png)

**Comment by jaredpar** [comment]
I think this is likely the result of VsVim and another plugin interacting badly.  Outside VsVim, Navigate To is actually one of my favorite features.  If VsVim always broke it then I would've fixed this long ago :smile: 

Can you tell me what other plugins you have installed here?

**Comment by dnperfors** [comment]
Sorry for the late reply, I just went on holiday when you wrote it.
I found the plugin that caused this problem, it was ReSharper. I deleted ReSharper since I didn't use it anymore, and now it works. Other plugins I have installed don't have any problems with VsVim (as far as I can find):

- Relative Number
- EditorConfig
- PowerTools 2015
- Refactoring Essentials
- CodeLens code health indicator
- etc.
